google/resource_compute_region_instance_group_manager_test.go

```diff
@@ -195,53 +163,17 @@ func TestAccRegionInstanceGroupManager_rollingUpdatePolicy(t *testing.T) {
 		Steps: []resource.TestStep{
 			resource.TestStep{
 				Config: testAccRegionInstanceGroupManager_rollingUpdatePolicy(igm),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckRegionInstanceGroupManagerBetaExists(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", &manager),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "update_strategy", "ROLLING_UPDATE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.type", "PROACTIVE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.minimal_action", "REPLACE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.max_surge_fixed", "2"),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.max_unavailable_fixed", "2"),
-					resource.TestCheckResourceAttr(
-						"google_compute_region_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.min_ready_sec", "20"),
-				),
 			},
+			// No import step because rolling updates are broken and the field will be removed in 2.0.0.
```

Comment:
Can we get a TODO on this to track its removal?

Reply:
Done